src/previewFrame/elementHighlighterScript.js

```diff
@@ -0,0 +1,36 @@
+const elementHighlighterScript = `(${function() {
+  window.addEventListener('message', (message) => {
+    const data = JSON.parse(message.data);
+    const {selector: {selector: selectorFromMessage}} = data;
+    const selector = selectorFromMessage;
+    updateCovers(selector);
+  });
+
+  function updateCovers(selector) {
+    const highlighterElements =
+    window.document.querySelectorAll('.popcode_HIGHLIGHTER');
+    if (highlighterElements.length !== 0) {
+      highlighterElements.forEach((highlighterElement) => {
+        highlighterElement.remove();
+      });
+    }
+
```

Comment:
Didn't include scroll or resizing handling / throttling this time around.  Removing highlights when editor not focused will remove the need for a scroll handler. Trying to find ways to prevent this from happening....

![screen shot 2017-09-20 at 4 03 04 pm](https://user-images.githubusercontent.com/9774128/30766976-d9fdd030-9fc6-11e7-99ef-7350a4b1c449.png)

Reply:
How does one cause that to happen?

Reply:
Since the positions of the highlighted elements are fixed,  when the screen is scrolled the elements move down screen but the highlighter elements remain fixed.